jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -135,7 +135,8 @@ private static void assertExtraDirectoryDeprecationWarning(String pomXml)
     BuildResult buildResult =
         JibRunHelper.buildToDockerDaemon(simpleTestProject, targetImage, pomXml);
     Assert.assertEquals(
-        "Hello, world. \nrw-r--r--\nrw-r--r--\nfoo\ncat\n",
+        "Hello, world. \n1970-01-01T00:00:01Z\nrw-r--r--\nrw-r--r--\nfoo\ncat\n"
```

Comment:
The output for this test program is starting to get more and more complicated, I wonder if we should think about making it more readable (not this PR)

Reply:
I felt the same. It's growing duplicates unnecessarily.